sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
I think the rules must be common and if we require blueprints for such changes, this changes must not be exception. I created blueprints, can be found here: https://blueprints.launchpad.net/percona-server/+spec/lock-binlog-for-backup-expose-5.6, https://blueprints.launchpad.net/percona-server/+spec/lock-binlog-for-backup-expose-5.7.